From 8ad252ab40b8f95db8413220146d54bf8a7d7be8 Mon Sep 17 00:00:00 2001
From: John Daley <johndale@cisco.com>
Date: Fri, 18 Mar 2016 11:27:07 -0700
Subject: [PATCH 19/22]     enic: update maintainers

    Change maintainers for ENIC PMD and fix pointer to enic
    documentation in MAINTAINERS.

    Signed-off-by: John Daley <johndale@cisco.com>
---
 MAINTAINERS              | 3 ++-
 doc/guides/nics/enic.rst | 2 +-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index b90aeea..f5b8bb4 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -264,8 +264,9 @@ F: doc/guides/nics/cxgbe.rst
 
 Cisco enic
 M: John Daley <johndale@cisco.com>
-M: Sujith Sankar <ssujith@cisco.com>
+M: Nelson Escobar <neescoba@cisco.com>
 F: drivers/net/enic/
+F: doc/guides/nics/enic.rst
 
 Combo szedata2
 M: Matej Vido <matejvido@gmail.com>
diff --git a/doc/guides/nics/enic.rst b/doc/guides/nics/enic.rst
index 2a228fd..e67c3db 100644
--- a/doc/guides/nics/enic.rst
+++ b/doc/guides/nics/enic.rst
@@ -218,4 +218,4 @@ Any questions or bugs should be reported to DPDK community and to the ENIC PMD
 maintainers:
 
 - John Daley <johndale@cisco.com>
-- Sujith Sankar <ssujith@cisco.com>
+- Nelson Escobar <neescoba@cisco.com>
-- 
1.9.1

